bmEditButtonPopup.lua

```diff
@@ -129,8 +128,23 @@ function BMButtonEditor:RenderEditButtonPopup()
             picker:SetClosed()
             self:CloseEditPopup()
         end
-    end
 
+    end
+    if ImGui.IsWindowHovered(ImGuiHoveredFlags.ChildWindows)  then

```

Comment:
Shouldnt this jsut call the same code as the save button? feels like all the saves should go through 1 code path.